database/generateData.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const faker = require('faker');
+
+/*
+TO SEED DATA IN MONGO:
+  mongoimport --db apateez-gallery --collection Places --file allData.json
+  --jsonArray --numInsertionWorkers 10 | gnomon
+
+TO SEED DATA IN POSTGRES FROM WITHIN apateez DB:
+  \copy gallery ("place_id", "name", "photos")
+  FROM '/Users/Pooja_Kodavanti/Desktop/HR-SDC/gallery/database/allData.csv'
+  DELIMITER ',' CSV HEADER;
+*/
+
+// Specify command line arguments
+const args = process.argv.slice(2);
+const genJSON = args.includes('--json'); // if included in bash script, create json
+const numEntries = args.find(item => /\b\d+\b/.test(item)) || 10; // if there is num in bash script, use as entry val
+const PHOTO_COUNT = 992;
+
+const genRandPhotos = () => Array(10).fill(0).map(() => Math.floor(Math.random() * PHOTO_COUNT));
+
+const buildEntryJSON = (id) => {
+  const obj = {
+    place_id: id,
+    name: `${faker.company.companyName()}`,
+    photos: genRandPhotos(),
+  };
+  return JSON.stringify(obj);
+};
+
+const buildEntryCSV = id => `${id},"${faker.company.companyName()}","{${genRandPhotos()}}"\n`;
+
+const generateData = () => {
+  const options = {
+    autoClose: true,
+  };
+  const writeStream = genJSON ? fs.createWriteStream('allData.json', options) : fs.createWriteStream('allData.csv', options);
+  let i = 0;
+  const write = () => {
+    let ok = true;
+    do {
+      i += 1;
+      if (i === 1) {
+        writeStream.write(genJSON ? `[${buildEntryJSON(i)},` : `${buildEntryCSV(i)}`);
```

Comment:
you might want to do a similar ternary to line 28 where you story the correct buildEntry function and then just reference that throughout these lines.